### PR TITLE
Implemented Jstyle file filter and opening in last dir

### PR DESCRIPTION
**Pull request by Siedlerchr**
Filter filter for jstyle files #1294 
Extended BrowseAction to display file filter
Fixed NPE when selecting invalid jstyle (prevented style dialog from opening)

- [X ] Change in CHANGELOG.md described

### CODE REVIEW

File: src/main/java/net/sf/jabref/gui/actions/BrowseAction.java [hunk 1]
```diff
@@ -38,33 +38,35 @@
...
     }
 
     public static BrowseAction buildForDir(JTextField tc) {
-        return new BrowseAction(null, tc, true, null);
+        return new BrowseAction(null, tc, true, null, Globals.NONE);
     }
 
     public static BrowseAction buildForFile(JTextField tc) {
-        return new BrowseAction(null, tc, false, null);
+        return new BrowseAction(null, tc, false, null, Globals.NONE);
```
Comment (simonharrer) [inline src/main/java/net/sf/jabref/gui/actions/BrowseAction.java]: Why Globals.NONE? Why not simply an empty string? Seems to be an unecessary dependency.

**Comment by simonharrer** [comment]
LGTM

File: src/main/java/net/sf/jabref/gui/actions/BrowseAction.java (hunk 1 above)
Comment (Siedlerchr) [inline src/main/java/net/sf/jabref/gui/actions/BrowseAction.java]: Yes, you are right. Interestingly the Globals.NONE was used before. I will look into that this evening.

**Comment by Siedlerchr** [comment]
@simonharrer Would you merge this in?

**Pull request merged**